main.js

```diff
@@ -1 +1,10 @@
-
+function toggleToDo(element){
+  var newClass;
+  var completed = element.querySelector(".fa").classList.contains("fa-check-square");
```

Comment:
@ChrisBaidoo, could you refactor so we don't have to duplicate `element.querySelector(".fa")` 4 times.

Reply:
@samdemaeyer I have created a var for the `element.querySelector(".fa")`  called faClassName 

`var faClassName = element.querySelector(".fa")`